node/network/protocol/src/peer_set.rs

```diff
@@ -36,9 +36,17 @@ impl PeerSet {
 	/// network service.
 	pub fn get_info(self) -> NonDefaultSetConfig {
 		let protocol = self.into_protocol_name();
+		let max_notification_size = match self {
+			PeerSet::Validation => 1024 * 1024,
```

Comment:
`PoV` are also sent on the `ValidationProtocol`, so message limits should be the same IMO